### ISSUE TITLE
Documentation on using a separate project for migrations not very clear and hard to follow

### ISSUE THREAD

**Issue by whizkidwwe1217** [aspnet/EntityFramework.Docs#846]
Please make the documentation a bit clearer and provide a full sample code.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b545eeff-6c33-3cc0-09ce-a3ad342d7438
* Version Independent ID: 34bc1bac-7338-d8ab-390d-8c5e88a48c10
* Content: [Migrations with Multiple Projects - EF Core](https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/projects)
* Content Source: [entity-framework/core/managing-schemas/migrations/projects.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/managing-schemas/migrations/projects.md)
* Product: **entity-framework**
* GitHub Login: @bricelam
* Microsoft Alias: **bricelam**

**Comment by ajcvickers** [comment]
@whizkidwwe1217 Can you provide some details as to where it was unclear and/or hard to follow?

**Comment by whizkidwwe1217** [comment]
@ajcvickers I have a hard time understanding step 5. Does it mean I should reference the Migrations assembly to the Startup project?

**Comment by ajcvickers** [comment]
@whizkidwwe1217 Yes.

**Comment by footysteve** [comment]
Hi, step 5 really isn't very clear. When I read it I thought won't that create a circular ref, then read a little further and that is mentioned with a suggested fix.

However, that process always creates a circular ref which in vs community at least, isn't possible - an error dialog informs us of this.

Applying the fix seems to work, at least if you leave the model classes in the original project which defeats the object, but still a circular ref cannot be added which is confusing because the docs suggest this is required. If you move your model to the class library the need for that ref becomes apparent.

And there is no way round that, the project won't compile without the circular ref which can't be added.

Or did I not I miss something.

Either way it's confusing.

Thanks
Steve
